pandas/tests/reshape/test_concat.py

```diff
@@ -2739,3 +2739,13 @@ def test_concat_empty_df_object_dtype():
     result = pd.concat([df_1, df_2], axis=0)
     expected = df_1.astype(object)
     tm.assert_frame_equal(result, expected)
+
+
+def test_concat_sparse():
+    # GH 23557
+    a = pd.Series(pd.SparseArray([0, 1, 2]))
```

Comment:
can you add
```
from pandas.array import SparseArrary
```

at the top instead of directly using SparseArray as we are doing to deprecate that import.

Reply:
Done :slightly_smiling_face: